covid_graphs/covid_graphs/wsgi.py

```diff
@@ -0,0 +1,10 @@
+from os import getenv
+from pathlib import Path
+
+from covid_graphs.scripts.server import uswgi_server
+
+data_path = getenv(key="DATA_PATH", default="data")
+app = uswgi_server(Path(data_path))
+
+if __name__ == "__main__":
+    app.run()
```

Comment:
The files associated with the web should be moved elsewhere, e.g. under `/web` in the root of the repository. Also paging @miskosz for an opinion.

I think our Python package should end at providing a Dash or Flask app and then the web server would run it using WSGI or whatever. The WSGI parts should be inside the `/web` directory.

Reply:
Rearrenged with your help.